packages/xstate-angular/src/injectActor.ts

```diff
@@ -0,0 +1,57 @@
+import { createActor } from 'xstate';
+import type {
+  ActorOptions,
+  ActorRefFrom,
+  AnyActorLogic,
+  SnapshotFrom
+} from 'xstate';
+import {
+  DestroyRef,
+  Injector,
+  inject,
+  runInInjectionContext,
+  signal,
+  Signal
+} from '@angular/core';
+
+interface injectActorOptions {
+  injector?: Injector;
+}
+
+interface InjectedActor<TLogic extends AnyActorLogic> {
+  snapshot: Signal<SnapshotFrom<TLogic>>;
+  send: ActorRefFrom<TLogic>['send'];
+  ref: ActorRefFrom<TLogic>;
+}
+
+export function injectActor<TLogic extends AnyActorLogic>(
+  logic: TLogic,
+  options?: ActorOptions<TLogic>,
+  injectOptions?: injectActorOptions
```

Comment:
I'd probably merge those two:
```suggestion
  { injector = inject(Injector), ...options }: ActorOptions<TLogic> & { injector?: Injector } = {},
```